src/kili/core/authentication.py

```diff
@@ -23,12 +24,23 @@ class KiliAuth:
     def __init__(
         self, api_key: str, api_endpoint: str, client_name: GraphQLClientName, verify=True
     ) -> None:
+        """Initialize KiliAuth.
+
+        Args:
+            api_key: Kili API key.
+            api_endpoint: Kili API endpoint.
+            client_name: Name of the client.
+            verify: Whether to verify the SSL certificate.
+        """
         self.api_key = api_key
         self.api_endpoint = api_endpoint
         self.client_name = client_name
         self.verify = verify
 
-        self.check_api_key_valid()
+        skip_checks = os.environ.get("KILI_SDK_SKIP_CHECKS", None) is not None
```

Comment:
maybe that data quality can set this environment variable in the env that runs the plugin in google ?

Reply:
yep! It should improve the init delay

Reply:
@p-desaintchamas @gheorghetutunaru